pandas/tests/scalar/interval/test_arithmetic.py

```diff
@@ -45,3 +45,14 @@ def test_numeric_interval_add_timedelta_raises(interval, delta):
 
     with pytest.raises((TypeError, ValueError), match=msg):
         delta + interval
+
+
+def test_timdelta_add_timestamp_interval():
+    delta = np.timedelta64(0)
```

Comment:
here or follow-up we should get the full complement of add/sub with dt64/td64/pytimedelta/Tick/...

Reply:
Might as well parametrize over a few timedelta types here